fiftyone/utils/sam2.py

```
@@ -32,89 +32,76 @@
-    fout.TorchImageModelConfig, fozm.HasZooModel
-):
-    """Configuration for running a :class:`SegmentAnything2VideoModel`.
-
-    See :class:`fiftyone.utils.torch.TorchImageModelConfig` for additional
-    arguments.
-    """
 
-    def __init__(self, d):
-        d = self.init(d)
-        super().__init__(d)
+    def point_transform(self, points, img_hw, point_labels=None):
```

Ruff (0.15.5)

[warning] Missing return type annotation for private function `point_transform`

(ANN202)